napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -4,6 +4,7 @@
 from typing import Callable, Dict, List, Sequence, Tuple
 
 from napari_plugin_engine.dist import standard_metadata
```

Comment:
it would be nice to get rid of this function too, this is a relatively unnecessary layer on top of `importlib.metadata.metadata()`, so instead of:

```
from napari_plugin_engine.dist import standard_metadata

standard_metadata('napari').get('version')

# you can do:
try:
    from importlib.metadata import metadata
except ImportError:
    from importlib_metadata import metadata

metadata("napari").get("version")
```

note that the former will raise an exception if the distname doesn't exist, whereas the latter just returns an empty dict.  So might need to check the values passed on 698 in `qt_plugin_dialog`